applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.h

```diff
@@ -16,6 +16,8 @@
 // Project includes
 #include "containers/array_1d.h"
 #include "includes/ublas_interface.h"
+#include "containers/global_pointers_vector.h"
+#include "utilities/geometry_utilities.h"
```

Comment:
I would rather use forward declaration here to reduce compile time:

```suggestion

namespace Kratos
{
    // forward-declaring
    class Element;
    class ProcessInfo;
    class ModelPart;
    template< class TDataType >
    class GlobalPointersVector;
    template<class TDataType>
    class Dof;
    template<std::size_t TDimension, class TDofType>
    class Node;
    template<class TPointType>
    class Geometry;
```